archunit-junit/junit5/engine/src/test/java/com/tngtech/archunit/junit/internal/ArchUnitTestEngineTest.java

```diff
@@ -725,6 +726,42 @@ void filtering_included_packages() {
                     simpleRulesId(engineId));
         }
 
+        @Test
+        void filtering_specific_rule_by_system_property() {
+            System.setProperty("archunit.junit.testFilter", SimpleRules.SIMPLE_RULE_FIELD_TWO_NAME);
+
+            EngineDiscoveryTestRequest discoveryRequest = new EngineDiscoveryTestRequest()
+                    .withPackage(SimpleRuleLibrary.class.getPackage().getName())
+                    .withPackageNameFilter(excludePackageNames(WrongRuleMethodNotStatic.class.getPackage().getName()));
+
+            TestDescriptor rootDescriptor = testEngine.discover(discoveryRequest, engineId);
+
+            Set<UniqueId> leafIds = getAllLeafUniqueIds(rootDescriptor);
+            assertThat(leafIds).isNotEmpty();
+            leafIds.forEach(leafId -> {
+                assertThat(leafId.getLastSegment().getType()).isEqualTo(FIELD_SEGMENT_TYPE);
+                assertThat(leafId.getLastSegment().getValue()).isEqualTo(SimpleRules.SIMPLE_RULE_FIELD_TWO_NAME);
+            });
+        }
+
+        @Test
+        void filtering_specific_rules_by_system_property() {
+            System.setProperty("archunit.junit.testFilter", "simple_rule_method_two,simple_rule");
```

Comment:
We could add another filter for a rule that does not exist in order to show that this does not break anything:
```suggestion
            System.setProperty("archunit.junit.testFilter", "simple_rule_method_two,simple_rule,non-existing-rule");
```